### PR TITLE
Update train_main.py

### PR DESCRIPTION
**Pull request by woldier**
之前的版本中, 测试集中计算所有样本的平均MSE时, 用到的batch大小为训练集的batch 大小, 因此导致计算出来的MSE很低. In the previous version, when calculating the average MSE of all the samples in the test set, the batch size used was that of the training set, which resulted in a very low MSE.

### CODE REVIEW

**Comment by tonyckc** [comment]
Thanks for your update.

**Review by tonyckc — COMMENTED** [review]
Typo about the number of testing samples.
